oat-sa/tao-test-runner-qti-fe
# [TCA-525] Tools - Timer - Add Accessibility

Related to : https://oat-sa.atlassian.net/browse/TCA-525
 
Add invisible timers available for screen readers to H1 heading.
Introduce timer warning notification for screen readers
  
#### How to test
 
- prepare an instance of TAO with taoAct extension
- prepare a test with time constraints on test, part, section and item scope
- execute delivery as a test taker
- make sure that timers duplicated in h1 heading
- make sure that timers notifications announced by screen reader

#### Dependencies
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-testqti/pull/1770
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1358
